index.js

```diff
@@ -268,6 +276,10 @@ HtmlWebpackIncludeAssetsPlugin.prototype.apply = function (compiler) {
         includeAssetCount = includeAssetPaths.length;
         for (var a = 0; a < includeAssetCount; a++) {
           includeAssetPath = includeAssetPaths[a];
+
+          // var fsContent = fs.readFileSync(includeAssetPath);
```

Comment:
Please remove commented out code :-)

This kind of code would be best to include in a new unit test...